File: sdkwork-ai-sdk-go/types/completion_tokens_details.go
package types

// 输出token详情
type CompletionTokensDetails struct {
	ReasoningTokens int `json:"reasoning_tokens"`
	AudioTokens int `json:"audio_tokens"`
	AcceptedPredictionTokens int `json:"accepted_prediction_tokens"`
	RejectedPredictionTokens int `json:"rejected_prediction_tokens"`
}

File: sdkwork-ai-sdk-typescript/src/types/completion-tokens-details.ts
/** 输出token详情 */
export interface CompletionTokensDetails {
  /** 思维链token数量 */
  reasoning_tokens?: number;
  /** 音频token数量 */
  audio_tokens?: number;
  /** 接受预测token数量 */
  accepted_prediction_tokens?: number;
  /** 拒绝预测token数量 */
  rejected_prediction_tokens?: number;
}
